videos/videoModel.js

```diff
@@ -1,26 +1,36 @@
 const db = require("../database/dbConfig.js");
 
 module.exports = {
-  find,
-  findById,
-  findByUserId,
+    find,
+    findById,
+    findByUserId,
 };
 
 function find() {
-  return db("videos").select("*");
+    return db("videos").select("*");
 }
 
-function findById(id) {
-  return db
-    .select("*")
-    .from("videos")
-    .where({ id: id })
-    .first();
+function findById(video_id) {
+    return db("videos")
+        .join("users", "videos.owner_id", "users.id")
+        .join("prompts", "videos.prompt_id", "prompts.id")
+        .where("videos.id", video_id)
+        .first()
+        .select(
+            "videos.owner_id",
```

Comment:
I would add an "as" to these to make the data returned from the api cleaner. So instead of

`select(
"videos.owner_id"
...etc.
)
`

I would do:

`select(
"videos.owner_id as owner_id"
...etc.
)
`